recipes/rise/post-link.sh

```diff
@@ -0,0 +1 @@
+"${PREFIX}/bin/jupyter-nbextension" enable rise --py --sys-prefix
```

Comment:
Not for now but when https://github.com/damianavila/RISE/pull/342 lands, we should modify and delete this post steps, because there will be no needed anymore...